samples/openthread/cli/README.rst

```diff
@@ -18,6 +18,9 @@ This sample has Commissioner, Joiner, and Border Router roles enabled by default
 If used alone, the sample allows you to test the network status.
 It is recommended to use at least two development kits running the same sample to be able to test communication.
 
+An additional functionality of this particular sample is the possibility of updating OpenThread libraries
+with the version build from the current source.
```

Comment:
Mention an option for doing that as well

Reply:
done